### PR TITLE
Fix SSDP discovery for MacOS

### PR DESCRIPTION
**Pull request by jphalip**

### CODE REVIEW

File: src/main/java/org/jf/fusionIdea/run/FusionScriptState.java [hunk 1]
```diff
@@ -236,40 +242,15 @@ private MulticastSocket sendIpv4SSDPRequest() {
...
-                    }
-                }
-
-                if (!success) {
-                    FusionIdeaPlugin.log.warn("Couldn't send ipv6 ssdp packet on any interface");
-                    return null;
-                }
-
+                InetAddress LOCALHOST_IPV6 = Inet6Address.getByName("::1");
+                MulticastSocket socket = new MulticastSocket(new InetSocketAddress(LOCALHOST_IPV6, 0));
```
Comment (JesusFreke) [inline src/main/java/org/jf/fusionIdea/run/FusionScriptState.java]: This works on windows if I don't specify an interface for this at all. e.g. 

```java
                MulticastSocket socket = new MulticastSocket();
                socket.setLoopbackMode(false);
                DatagramPacket packet = new DatagramPacket(
                    SEARCH_MESSAGE, SEARCH_MESSAGE.length,
                    new InetSocketAddress(
                        Inet6Address.getByName(MULTICAST_GROUP_IPV6), MULTICAST_PORT));
```

Comment (JesusFreke) [reply src/main/java/org/jf/fusionIdea/run/FusionScriptState.java]: See also my comments in the other PR.

If the above change doesn't work for mac, we'll probably need to do a platform check here too.

Comment (jphalip) [reply src/main/java/org/jf/fusionIdea/run/FusionScriptState.java]: I added the platform check. Somehow on the Mac it appears to be necessary to both set the socket on localhost and to explicitly set the interface.

**Comment by JesusFreke** [comment]
Looks good. I confirmed both IPv4 and IPv6 are working on Windows as well. thanks!

**Pull request merged**